src/test/TestDASHTree.cpp

```diff
@@ -15,6 +15,7 @@
 #include "../CompKodiProps.h"
 
 #include <gtest/gtest.h>
+#include <decrypters/Helpers.h>
```

Comment:
and moved above by sorting it
```suggestion
#include "../decrypters/Helpers.h"
```